src/services/compiler/host.ts

```diff
@@ -114,14 +130,98 @@ export default class CompilerHost extends AsyncEventEmitter implements CompilerP
         ], this);
     }
 
+    private _setupDebuggerHandlers (): void {
+        if (!this.cdp)
+            return;
+
+        testRunTracker.on(DEBUG_ACTION.resume, async () => {
+            if (!this.cdp)
+                return;
+
+            const disableDebugMethodName = TestController.disableDebugForNonDebugCommands.name;
+
+            // NOTE: disable `debugger` for non-debug commands if the `Resume` button is clicked
+            // the `includeCommandLineAPI` option allows to use the `require` functoion in the expression
+            await this.cdp.Runtime.evaluate({
+                expression:            `require.main.require('../../api/test-controller').${disableDebugMethodName}()`,
```

Comment:
I feel we should use absolute paths here. Otherwise we can get some problems with custom runners.

```suggestion
                //TODO: refactor to use absolute paths
                expression:            `require.main.require('../../api/test-controller').${disableDebugMethodName}()`,
```